### PR TITLE
Add Auth0, Micro User, Stytch and create a new section Authentication

### PR DESCRIPTION
**Pull request by asim**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by asim** [comment]
Still to add 2 more in the new "Authentication" category

**Review by fintzd — CHANGES_REQUESTED** [review]
Hey! This is not an API interface though is it?

**Review by fintzd — APPROVED** [review]
My mistake, it's all good!

**Comment by asim** [comment]
Everything is now done and ready to be merged.

**Comment by asim** [comment]
Has been squashed. Any timeline on a merge?

File: README.md [hunk 1]
```diff
@@ -121,6 +122,13 @@ API | Description | Auth | HTTPS | CORS |
 | [Pixel Encounter](https://pixelencounter.com/api) | SVG Icon Generator | No | Yes | No |
 | [Rijksmuseum](https://www.rijksmuseum.nl/en/api) | Art | `apiKey` | Yes | Unknown |
 
+### Authentication
+API | Description | Auth | HTTPS | CORS |
+|---|---|---|---|---|
+| [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
+| [Micro User Service](https://m3o.com/user) | User management and authentication | `apiKey` | Yes | No |
+| [Stytch](https://stytch.com/) | User infrastructure for modern applications | `apiKey` | Yes | No |
+
```
Comment (matheusfelipeog) [inline README.md:131]: Add index anchor (`**[⬆ Back to Index](#index)**`) before section name

```suggestion
**[⬆ Back to Index](#index)**
### Authentication
API | Description | Auth | HTTPS | CORS |
|---|---|---|---|---|
| [Auth0](https://auth0.com) | Easy to implement, adaptable authentication and authorization platform | `apiKey` | Yes | Yes |
| [Micro User Service](https://m3o.com/user) | User management and authentication | `apiKey` | Yes | No |
| [Stytch](https://stytch.com/) | User infrastructure for modern applications | `apiKey` | Yes | No |

```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @asim, we are grateful for that <3

All APIs is OK ✅

Only a anchor is missing from the markdown structure.

Can you resolve this review, please?

**Comment by asim** [comment]
Thanks very much. I'll make that change shortly ☺️

File: README.md (hunk 1 above)
Comment (asim) [reply README.md:132]: The back to index link has now been added

**Comment by asim** [comment]
Back to index anchor added. Please review and if all good merge :)

**Review by fintzd — APPROVED** [review]
Looks all good

**Review by matheusfelipeog — APPROVED** [review]
Perfect ;)

A big THANKS for solving this @asim ❤️

**Comment by asim** [comment]
Thank you for accepting the submission :)

**Pull request merged**